sami2py/_core_class.py

```diff
@@ -3,52 +3,45 @@
 # Copyright (C) 2017, JK & JH
 # Full license can be found in License.md
 # -----------------------------------------------------------------------------
-"""Wrapper for running sami2 model
+"""Wrapper for running sami2 model.
+
 Classes
 -------
 Model
     Loads, reshapes, and holds SAMI2 output for a given model run
     specified by the user.
 
-Moduleauthor
-------------
-Jeff Klenzing (JK), 1 Dec 2017, Goddard Space Flight Center (GSFC)
-
 """
 
 import numpy as np
 from os import path
 import re
-import warnings
 
 import xarray as xr
-import matplotlib.pyplot as plt
 
-from sami2py.utils import generate_path, get_unformatted_data, return_fourier
+import sami2py
 
 
 class Model(object):
-    """Python object to handle SAMI2 model output data
-    """
+    """Python object to handle SAMI2 model output data."""
 
     def __init__(self, tag, lon, year, day, outn=False, test=False):
-        """ Loads a previously run sami2 model and sorts into
-            appropriate array shapes
+        """Load a previously run sami2 model and sorts into array shapes.
```

Comment:
```suggestion
        """Load a previously run SAMI2 model output and sorts into array shapes.
```

Here and elsewhere capitalize SAMI2.